src/transport/grpc/mod.rs

```diff
@@ -243,18 +244,27 @@ impl Receive for Grpc {
         _mapping: MappingRef<'_, &Interface>,
         payload: Self::Payload,
     ) -> Result<(AstarteType, Option<Timestamp>), crate::Error> {
-        let astarte_data_type::Data::AstarteIndividual(individual_data) = payload.data else {
-            return Err(crate::Error::from(
-                GrpcTransportError::DeserializationExpectedIndividual,
-            ));
+        let data = match payload.data {
+            Payload::AstarteData(data) => data,
+            Payload::AstarteUnset(astarte_message_hub_proto::AstarteUnset {}) => {
+                trace!("unset received");
```

Comment:
Do we want to keep it for debugging purposes?

Reply:
Yes, is also lower than the usual RUST_LOG=debug that we have in other CIs. So it should be disabled in most cases